ssc/cmod_tcsmolten_salt.cpp

```diff
@@ -492,15 +494,23 @@ static var_info _cm_vtab_tcsmolten_salt[] = {
+    { SSC_OUTPUT,    SSC_ARRAY,  "T_tes_hot",                          "TES hot temperature",                 "C",            "",                                  "",            "*",  "",  "" },
+    { SSC_OUTPUT,    SSC_ARRAY,  "T_tes_cold",                         "TES cold temperature",                "C",            "",                                  "",            "*",  "",  "" },
+    { SSC_OUTPUT,   SSC_ARRAY,   "mass_tes_cold",                      "TES cold tank mass (end)",            "kg",           "",                                  "",            "*",  "",  "" },
+    { SSC_OUTPUT,   SSC_ARRAY,   "mass_tes_hot",                       "TES hot tank mass (end)",             "kg",           "",                                  "",            "*",  "",  "" },
+    { SSC_OUTPUT,    SSC_ARRAY,  "q_dc_tes",                           "TES discharge thermal power",         "MWt",          "",                                  "",            "*",  "",  "" },
+    { SSC_OUTPUT,    SSC_ARRAY,  "q_ch_tes",                           "TES charge thermal power",            "MWt",          "",                                  "",            "*",  "",  "" },
+    { SSC_OUTPUT,    SSC_ARRAY,  "e_ch_tes",                           "TES charge state",                    "MWht",         "",                                  "",            "*",  "",  "" },
+    { SSC_OUTPUT,    SSC_ARRAY,  "m_dot_cr_to_tes_hot",                "Mass flow: field to hot TES",         "kg/s",         "",                                  "",            "*",  "",  "" },
+    { SSC_OUTPUT,    SSC_ARRAY,  "m_dot_tes_hot_out",                  "Mass flow: TES hot out",              "kg/s",         "",                                  "",            "*",  "",  "" },
+    { SSC_OUTPUT,    SSC_ARRAY,  "m_dot_pc_to_tes_cold",               "Mass flow: cycle to cold TES",        "kg/s",         "",                                  "",            "*",  "",  "" },
+    { SSC_OUTPUT,    SSC_ARRAY,  "m_dot_tes_cold_out",                 "Mass flow: TES cold out",             "kg/s",         "",                                  "",            "*",  "",  "" },
+    { SSC_OUTPUT,    SSC_ARRAY,  "m_dot_field_to_cycle",               "Mass flow: field to cycle",           "kg/s",         "",                                  "",            "*",  "",  "" },
+    { SSC_OUTPUT,    SSC_ARRAY,  "m_dot_cycle_to_field",               "Mass flow: cycle to field",           "kg/s",         "",                                  "",            "*",  "",  "" },
+    //{ SSC_OUTPUT,    SSC_ARRAY,  "m_dot_tes_dc",                       "TES discharge mass flow rate",        "kg/s",         "",                                  "",          "*",  "",  "" },
+    //{ SSC_OUTPUT,    SSC_ARRAY,  "m_dot_tes_ch",                       "TES charge mass flow rate",           "kg/s",         "",                                  "",          "*",  "",  "" },
```

Comment:
Line up columns

Reply:
low priority and easier to understand variable if units are close to name and description